authentication.go

```diff
@@ -63,7 +63,7 @@ func (c *Client) RequestAppAccessToken(scopes []string) (*AppAccessTokenResponse
 		ClientSecret: opts.ClientSecret,
 		RedirectURI:  opts.RedirectURI,
 		GrantType:    "client_credentials",
-		Scopes:       scopes,
+		Scopes:       strings.Join(scopes, " "),
```

Comment:
I think I'd prefer the following:

```suggestion
		Scopes:       strings.Split(scopes, " "),
```